build-logic/binary-compatibility/src/main/groovy/gradlebuild/EnrichedReportRenderer.groovy

```diff
@@ -70,7 +70,15 @@ class EnrichedReportRenderer extends GroovyReportRenderer {
                         result.acceptedApiChanges.push(correction);
                     });
                     // Sort the array in place by type, then member
-                    result.acceptedApiChanges = result.acceptedApiChanges.sort((a, b) => (a.type +'#' + a.member) > (b.type + '#' + b.member));
+                    // Note that Firefox is fine with a sort function returning any positive or negative number, but Chrome 
+                    // requires 1 or -1 specifically and ignores higher or lower values
+                    result.acceptedApiChanges = result.acceptedApiChanges.sort((a, b) => { 
+                        if ((a.type +'#' + a.member) > (b.type + '#' + b.member)) {
+                            return 1; 
+                        } else {
+                            return -1;
+                        }
```

Comment:
🤔 You could also use `new Intl.Collator("ASCII").compare()` here. You can also avoid string concatenation if you do something like this:

```javascript
let result = compare(a.type, b.type);
if (result == 0) {
    result = compare(a.member, b.member);
}
return result;
```

Reply:
A good idea, but I want to keep the sorting algorithm consistent with that used by `AbstractAcceptedApiChangesMaintenanceTask`.  Which I also need to update now.